testing/e2e/e2e_api_test.go

```diff
@@ -21,14 +21,72 @@
 package e2e_test
 
 import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net"
+	"net/http"
+	"strings"
+	"time"
+
 	beaconapi "github.com/attestantio/go-eth2-client/api"
 	apiv1 "github.com/attestantio/go-eth2-client/api/v1"
 	"github.com/attestantio/go-eth2-client/spec/phase0"
 	"github.com/berachain/beacon-kit/node-api/handlers/utils"
 	"github.com/berachain/beacon-kit/testing/e2e/config"
 	"github.com/berachain/beacon-kit/testing/e2e/suite/types"
+	"github.com/ethereum/go-ethereum/common"
 )
 
+const localHost = "localhost"
+
+// StateValidatorResponse represents the API response for a validator.
+type StateValidatorResponse struct {
+	Data struct {
+		Index     string `json:"index"`
+		Balance   string `json:"balance"`
+		Status    string `json:"status"`
+		Validator struct {
+			Pubkey string `json:"pubkey"`
+		} `json:"validator"`
+	} `json:"data"`
+}
```

Comment:
@nidhi-singh02, question: why do we have to redefine it here?

Reply:
Addressed, removed it. Using `ValidatorData` from node-api package.